app/models/url.rb

```diff
@@ -2,8 +2,14 @@ class Url < ActiveRecord::Base
   validates :count, presence: true
   validates :original, uniqueness: true
 
-  def shorten
-    digest = Digest::SHA256.hexdigest(original)[0..5]
+  def shorten(host)
+    #digest = Digest::SHA256.hexdigest(original)[0..5]
+    #while Url.find_by(shortened: digest)
```

Comment:
Incorrect indentation detected (column 4 instead of 6).<br>Missing space after #.